designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/NodesApiEndpoints.scala

```diff
@@ -657,6 +652,78 @@ object NodesApiEndpoints {
           )
       )
 
+    val sourceCompilationExample: EndpointOutput.OneOfVariant[SourceCompilation] =
+      oneOfVariantFromMatchType(
+        BadRequest,
+        plainBody[SourceCompilation]
+          .example(
+            Example.of(
+              summary = Some("Source compilation error"),
+              value = SourceCompilation("sourceId", List("Invalid source configuration", "Missing required parameter"))
+            )
+          )
+      )
+
+    val unsupportedSourcePreviewExample: EndpointOutput.OneOfVariant[UnsupportedSourcePreview] =
+      oneOfVariantFromMatchType(
+        NotFound,
+        plainBody[UnsupportedSourcePreview]
+          .example(
+            Example.of(
+              summary = Some("Source preview not supported"),
+              value = UnsupportedSourcePreview("sourceId")
+            )
+          )
+      )
+
+    val noDataGeneratedExample: EndpointOutput.OneOfVariant[NoDataGenerated.type] =
+      oneOfVariantFromMatchType(
+        NotFound,
```

Comment:
More than one output variant with the same status code won't work correctly. There is a long discussion in Tapirs repo why it works that way. The unpleasant thing is that Tapir don't event report an Exception in that case. It justs ignore some of variants. To make it work with Tapir, you need to have one output variant with given status code with many examples. Take a look at for example `DeploymentApiEndpoints.runDeploymentEndpoint`